azext_iot/_help.py

```diff
@@ -576,6 +576,42 @@
         -m {module_name} -j ../mymodtwin.json
 """
 
+helps[
+    "iot hub state export"
+] = """
+    type: command
+    short-summary: Saves the configuration and device information of an IoT Hub to a file. 
+    examples:
+    - name: Export the devices in the specified hub to the specified file.
+      text: >
+        az iot hub state export -n {iothub_name} -f {filename}
+"""
+
+helps[
+    "iot hub state import"
+] = """
+    type: command
+    short-summary: Uploads the configuration and device information from a file to an IoT Hub.  
+    examples:
+    - name: Upload the devices from the specified file to the specified hub. 
+      text: >
+        az iot hub state import -n {iothub_name} -f {filename}
+    - name: Upload the devices from the default file to the specified hub, overwriting the configurations and devices already in the hub. 
+      text: >
+        az iot hub state import -n {iothub_name} --overwrite
```

Comment:
lets make an example file within docs/samples (search for Avin's adt-bulk-import-data-sample.ndjson)

Also if you have other sample files you want to help add, feel free to do so!